user_templates/variables.cfg

```diff
@@ -55,6 +55,7 @@ variable_probe_dock_margin_xy: 0, 0
 variable_safe_extruder_temp: 150
 
 ## Prime line parameters
+variable_use_adaptive_primeline: 1 # using the adaptive primeline manages start point and direction automatically
```

Comment:
I would change the name to "variable_prime_line_adaptive_mode" to be more aligned with the other variables and would also use True/False to stay aligned with all the other variables.cfg booleans